src/svgs/icons.tsv

```diff
@@ -84,6 +84,7 @@
 56		i_custom_emacs				emacs_nf.svg
 57		i_custom_orgmode			orgmode_nf.svg
 58		i_custom_kotlin				kotlin_nf.svg
+59		i_custom_asm				asm_nf.svg
```

Comment:
offset 59 is already given to ... see below `059 i_seti_apple`.
You need to add it in the bottom.

I see, the switch from two-digits to three-digits can be confusing, as is the comment line in between. I ask myself how they come to be there :grimacing: Must have been forgotten.

Reply:
Fixed